app/src/ui/app.tsx

```diff
@@ -1731,19 +1738,53 @@ export class App extends React.Component<IAppProps, IAppState> {
           return null
         }
 
+        const { repository, ourBranch, theirBranch } = popup
+        const submit = __DARWIN__ ? 'Commit Merge' : 'Commit merge'
+        const abort = __DARWIN__ ? 'Abort Merge' : 'Abort merge'
+        const headerTitle = (
+          <span>
+            {`Resolve conflicts before merging `}
+            {theirBranch !== undefined && <strong>{theirBranch}</strong>}
+            {` into `}
+            <strong>{ourBranch}</strong>
```

Comment:
Is it possible that `theirBranch` is actually `undefined`? If so the final text will have an extra space: `Resolve conflicts before merging  into`

Reply:
Good catch!. I updated it and here is what it looks like:
their undefined
![image](https://user-images.githubusercontent.com/75402236/119162285-2f4d8600-ba28-11eb-9774-1b80d6e466dd.png)

their not undefined
![image](https://user-images.githubusercontent.com/75402236/119161909-ca922b80-ba27-11eb-831a-7a446eb3171e.png)